src/wallet_setup/CreatePassword.tsx

```diff
@@ -62,7 +62,13 @@ const CreatePassword = function CreatePassword() {
 
   let passwordMatchElements: JSX.Element = (<div />);
   // handle additional state where the user's password doesn't meet requirements
-  if (passwordMatchState === 'match') {
+  if (password.length < 8) {
+    passwordMatchElements = (
+      <div id="create-password-match-elements">
+        <p id="create-password-info-mismatch" className="password-info text-danger">Your password is not at least 8 characters!</p>
```

Comment:
Consider using a positive rather than negative message, something like "Make sure your password is at least 8 characters!" if the password length is greater than 0 and less than 8.

If the password length is 0, don't show any error text (otherwise, when you first get to the Create Password page, it says your password isn't long enough even though you haven't started entering anything yet!)